config/application.rb

```diff
@@ -27,16 +27,9 @@ class Application < Rails::Application
       g.stylesheets false
     end
 
-    # https://github.com/plataformatec/devise/wiki/How-To:-Create-custom-layouts
     config.to_prepare do
-      [Devise::SessionsController,
-       Devise::RegistrationsController,
-       Devise::ConfirmationsController,
-       Devise::UnlocksController,
-       Devise::InvitationsController,
-       Devise::PasswordsController,
-       UserPasswordsController,].each do |views|
-        views.layout proc { |controller| Rails.env.development? && ENV["REDESIGN"] == "true" ? "application_redesign" : "authentication" }
+      if Rails.env.development? && ENV["REDESIGN"]
+        ApplicationController.layout "application_redesign"
```

Comment:
Nice.

Reply:
I think this is a good move for when we move the redesign to stage/prod. It allows us to easily remove the excess code.